views/articles/new.ejs

```diff
@@ -0,0 +1,48 @@
+<% include ../partials/header.ejs %>
+
+<nav class="navbar navbar-default" role="navigation">
+	<div class="container-fluid">
+		<div class="navbar-header">
+			<button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
+				<span class="sr-only">Toggle navigation</span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+			</button>
+			<a class="navbar-brand" href="#">The Daily Planet</a>
+		</div>
+		
+		<div class="collapse navbar-collapse navbar-ex1-collapse">
+			<ul class="nav navbar-nav navbar-right">
+				<div class="collapse navbar-collapse navbar-ex1-collapse">
+				<ul class="nav navbar-nav">
+					<li><a href="../about">About</a></li>
+					<li><a href="../articles">Articles</a></li>
+					<li><a href="../contact">Contact</a></li>
+					</ul>
+					<form class="navbar-form navbar-left" role="search">
+					<div class="form-group">
+						<input type="text" class="form-control" placeholder="Out Of Order">
+						<!-- Search does not work -->
+					</div>
+					<button type="submit" class="btn btn-default">Submit</button>
```

Comment:
Your form below wasn't working because I think you deleted the ```</form>``` closing tag for your search form here. Simply add ```</form>``` to line 29 and you should be able to post articles now.